### PR TITLE
Fixed frozen request when content-length equals 0

### PR DESCRIPTION
**Pull request by ar4s**
Prevent freezing request in `get_http_info`.

### CODE REVIEW

**Comment by ghost** [comment]
:ok:

File: src/ralph/discovery/http.py [hunk 1]
```diff
@@ -38,11 +54,12 @@ def get_http_info(ip):
         logger.error("Exception: {} for url: {}".format(e, url))
         url = "https://{}".format(':'.join((ip, '8006')))  # for Proxmox
         try:
-            response = requests.get(url, timeout=REQUEST_TIMEOUT, verify=False)
+            response = http_get_method(session, url)
             response.raise_for_status()
             headers, content = response.headers, response.content
         except requests.exceptions.RequestException as e:
             logger.error("Exception: {} for url: {}".format(e, url))
+    session.close()
```
Comment (mkurek) [inline src/ralph/discovery/http.py]: shouldn't this be in some finally block?

Comment (ar4s) [inline src/ralph/discovery/http.py]: maybe better than finally block will be with statement around whole `try..execpt` blocks?

**Comment by mkurek** [comment]
:ok:

**Pull request merged**